extensions/amp-wordpress-embed/1.0/amp-wordpress-embed.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseElement} from './base-element';
+import {dict} from '#core/types/object';
+import {isExperimentOn} from '#experiments';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-wordpress-embed';
+
+class AmpWordpressEmbed extends BaseElement {
```

Comment:
Dare I say, but should this not be `AmpWordPressEmbed` per [`capital_P_dangit`](https://developer.wordpress.org/reference/functions/capital_p_dangit/) law?

Reply:
Updated all `Wordpress` strings to `WordPress`.